### PR TITLE
2017 ioniq EV fingerprint

### PR DESCRIPTION
**Pull request by VictorLier**

### CODE REVIEW

File: selfdrive/car/hyundai/values.py [hunk 1]
```diff
@@ -199,14 +202,17 @@ class Buttons:
     (Ecu.fwdRadar, 0x7d0, None): [
       b'\xf1\x00AEev SCC F-CUP      1.00 1.00 96400-G7000         ',
       b'\xf1\x00AEev SCC F-CUP      1.00 1.00 96400-G7100         ',
+      b'\xf1\000AEev SCC F-CUP      1.00 1.00 96400-G7000         ',
```
Comment (adeebshihadeh) [inline selfdrive/car/hyundai/values.py:205]: which route did these versions come from?

File: selfdrive/car/hyundai/values.py [hunk 2]
```diff
@@ -137,6 +137,9 @@ class Buttons:
   }],
   CAR.IONIQ_EV_LTD: [{
     127: 8, 304: 8, 320: 8, 339: 8, 352: 8, 356: 4, 544: 7, 593: 8, 688: 5, 832: 8, 881: 8, 882: 8, 897: 8, 902: 8, 903: 8, 916: 8, 1040: 8, 1056: 8, 1057: 8, 1078: 4, 1136: 8, 1151: 6, 1168: 7, 1173: 8, 1265: 4, 1280: 1, 1287: 4, 1290: 8, 1291: 8, 1292: 8, 1294: 8, 1322: 8, 1342: 6, 1345: 8, 1348: 8, 1355: 8, 1363: 8, 1369: 8, 1407: 8, 1419: 8, 1425: 2, 1426: 8, 1427: 6, 1429: 8, 1430: 8, 1456: 4, 1470: 8, 1507: 8, 1535: 8
+  },
+  {
```
Comment (adeebshihadeh) [inline selfdrive/car/hyundai/values.py:141]: remove this. we don't upstream can fingerprints anymore

File: selfdrive/car/hyundai/values.py (hunk 1 above)
Comment (VictorLier) [reply selfdrive/car/hyundai/values.py:202]: From bc202a950a79fb6d|2021-09-19--17-22-13

File: selfdrive/car/hyundai/values.py [hunk 3]
```diff
@@ -199,14 +199,17 @@ class Buttons:
     (Ecu.fwdRadar, 0x7d0, None): [
       b'\xf1\x00AEev SCC F-CUP      1.00 1.00 96400-G7000         ',
       b'\xf1\x00AEev SCC F-CUP      1.00 1.00 96400-G7100         ',
+      b'\xf1\000AEev SCC F-CUP      1.00 1.00 96400-G7000         ',
     ],
     (Ecu.eps, 0x7d4, None): [
       b'\xf1\x00AE  MDPS C 1.00 1.02 56310G7300\x00 4AEEC102',
       b'\xf1\x00AE  MDPS C 1.00 1.04 56310/G7501 4AEEC104',
+      b'\xf1\000AE  MDPS C 1.00 1.03 56310/G7300 4AEEC103',
```
Comment (adeebshihadeh) [inline selfdrive/car/hyundai/values.py:207]:
```suggestion
      b'\xf1\x00AE  MDPS C 1.00 1.03 56310/G7300 4AEEC103',
```

File: selfdrive/car/hyundai/values.py [hunk 4]
```diff
@@ -199,14 +199,17 @@ class Buttons:
...
     ],
     (Ecu.fwdCamera, 0x7c4, None): [
       b'\xf1\x00AEE MFC  AT EUR LHD 1.00 1.00 95740-G7200 160418',
       b'\xf1\x00AEE MFC  AT USA LHD 1.00 1.00 95740-G2400 180222',
+      b'\xf1\000AEE MFC  AT EUR LHD 1.00 1.00 95740-G2300 170703',
```
Comment (adeebshihadeh) [inline selfdrive/car/hyundai/values.py:212]:
```suggestion
      b'\xf1\x00AEE MFC  AT EUR LHD 1.00 1.00 95740-G2300 170703',
```

File: selfdrive/car/hyundai/values.py (hunk 1 above)
Comment (adeebshihadeh) [reply selfdrive/car/hyundai/values.py:202]:
```suggestion
```

**Pull request merged**